flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/validation/JoinValidationTest.scala

```diff
@@ -136,4 +124,66 @@ class JoinValidationTest extends TableTestBase {
 
     streamUtil.verifySql(sql, "n/a")
   }
+
+  /** Rowtime attributes cannot be accessed in filter conditions yet. */
+  @Test(expected = classOf[TableException])
```

Comment:
please add also some trivial expected message matcher, otherwise this can give false negative results.